### ISSUE TITLE
Update @testing-library/user-event to latest version

### ISSUE THREAD

**Issue by frontend-specialisten** [Energinet-DataHub/greenforce-frontend#1988]
We want to update `@testing-library/user-event` to the latest version to migrate our tests to async / await and avoid flaky tests. 

**Acceptance criteria:**

- [ ] `@testing-library/user-event` is updated to the latest version 
- [ ] All existing tests are still passing

**Tech notes:**
This update may be blocked by `inputmask` or some other dependency. Start investigating that to have an overview of the size of this issue.

**Comment by ManBearTM** [comment]
Closing in favor of #2481
